e2e-tests/cypress/e2e/journeys/insurance/your-buyer/your-buyer.spec.js

```diff
@@ -0,0 +1,139 @@
+  cy.visit(`${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.YOUR_BUYER.YOUR_BUYER_DETAILS}`, {
+    auth: {
+      username: Cypress.config('basicAuthKey'),
+      password: Cypress.config('basicAuthSecret'),
+    },
+  });
+};
+
+context('Insurance - Your Buyer - Type of yourbuyer Page - As an exporter, I want to enter the type of Your Buyer details I need for my export contract', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.YOUR_BUYER.YOUR_BUYER_DETAILS}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  it('passes the audits', () => {
+    cy.lighthouse({
+      accessibility: 100,
+      performance: 75,
+      'best-practices': 100,
+      seo: 70,
+    });
+  });
+
+  it('should render a header with href to insurance start', () => {
+    partials.header.serviceName().should('have.attr', 'href', insuranceStartRoute);
+  });
+
+  it('renders a back link with correct url', () => {
+    partials.backLink().should('exist');
+    partials.backLink().invoke('text').then((text) => {
+      expect(text.trim()).equal(LINKS.BACK);
+    });
+
+    partials.backLink().click();
+
+    const expectedUrl = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.ALL_SECTIONS}`;
+
+    cy.url().should('eq', expectedUrl);
+
+    goToPageDirectly(referenceNumber);
+  });
+
+  it('renders an analytics cookies consent banner that can be accepted', () => {
+    cy.checkAnalyticsCookiesConsentAndAccept();
+  });
+
+  it('renders an analytics cookies consent banner that can be rejected', () => {
+    cy.rejectAnalyticsCookies();
+  });
+
+  it('renders a phase banner', () => {
+    cy.checkPhaseBanner();
+  });
+
+  it('renders a page title and heading with caption', () => {
+    const expectedPageTitle = `${CONTENT_STRINGS.PAGE_TITLE} - ${ORGANISATION}`;
+    cy.title().should('eq', expectedPageTitle);
+
+    headingCaption().invoke('text').then((text) => {
+      expect(text.trim()).equal(CONTENT_STRINGS.HEADING_CAPTION);
+    });
+
+    heading().invoke('text').then((text) => {
+      expect(text.trim()).equal(CONTENT_STRINGS.PAGE_TITLE);
+    });
+  });
+
+  it('renders an intro paragraph', () => {
+    insurance.policyAndExport.typeOfPolicyPage.intro().invoke('text').then((text) => {
+      expect(text.trim()).equal(CONTENT_STRINGS.INTRO);
+    });
+  });
```

Comment:
This is testing the type of policy page - needs to be only the your buyer page i.e a test that the new form field and label is rendered.

Will need to create a new yourBuyerPage in the E2E pages definition.

Then can remove the page intro and intro paragraph test